types/match-sorter/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for match-sorter 2.2
+// Type definitions for match-sorter 2.3
 // Project: https://github.com/kentcdodds/match-sorter#readme
 // Definitions by: Claas Ahlrichs <https://github.com/claasahl>
+//                 Christian Ruigrok <https://github.com/chrisru>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
I think we are just missing, the following:

```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
// TypeScript Version: 2.1
```